engine/src/main/java/org/terasology/rendering/opengl/OpenGLUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
Ah. Fair enough @Cervator. Then I'd just rename it OpenGLUtilities, at least for the time being.